homeassistant/components/withings/__init__.py

```diff
@@ -151,10 +158,14 @@ async def register_webhook(
             )
             return
 
+        webhook_name = "Withings"
+        if entry.title != DEFAULT_TITLE:
+            webhook_name += " ".join([webhook_name, entry.title])
```

Comment:
Won't this result in a name like `WithingsWithings TITLE`?

Reply:
You're right